keystrokes/src/components/Character/CharacterDetails/CharacterKeybindDisplay.js

```diff
@@ -9,16 +9,62 @@ import {
 	ExpansionPanelSummary,
 	ExpansionPanelDetails,
 } from '@material-ui/core';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+import ReactHtmlParser from 'react-html-parser';
+import update from 'immutability-helper';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import styleGuide from '../../../stylesheets/style';
 import { TabPanel } from '../helpers/TabPanels';
 
 CharacterKeybindDisplay.propTypes = {
 	index: PropTypes.number,
+	character: PropTypes.object,
+	setCharacter: PropTypes.func,
+	spec: PropTypes.number,
 	keyBinding: PropTypes.number,
 };
-function CharacterKeybindDisplay({ keyBinding, index }) {
+function CharacterKeybindDisplay({
+	keyBinding,
+	spec,
+	index,
+	character,
+	setCharacter,
+}) {
 	const classes = styleGuide();
+	const [descriptionToggle, setDescriptionToggle] = useState(false);
+	const [descriptionText, setDescriptionText] = useState(
+		character.specs[spec].keybindings[keyBinding][
+			Object.keys(character.specs[spec].keybindings[keyBinding])[0]
```

Comment:
this was the byproduct of that double key thing... not sure how to fix it so just implemented it with object.keys(...)[0]

Reply:
good question looks like it could be a function at the least get initial Description state?

Reply:
`character.specs[spec].keybindings[keyBinding][
			Object.keys(character.specs[spec].keybindings[keyBinding])[0]
].description || ''` 
this is really tricky to read, maybe worth breaking up into a few lines with variables of what each thing is. looks like at the end you're just getting description, why is it so hard?